airflow/example_dags/example_kubernetes_executor_config.py

```diff
@@ -52,59 +53,68 @@ def test_volume_mount():
     start_task = PythonOperator(
         task_id="start_task",
         python_callable=print_stuff,
-        executor_config={
-            "KubernetesExecutor": {
-                "annotations": {"test": "annotation"}
-            }
+        executor_config={"podOverride": k8s.V1Pod(
```

Comment:
After https://github.com/apache/airflow/pull/10756 is merged, we need to change `podOverride` -> `pod_override` in this PR